DataConnectors/CyberArk Data Connector.json

```diff
@@ -0,0 +1,88 @@
+{

```

Comment:
Please check in the logo file for data connectors per step 5 of [data connectors guidance](https://github.com/Azure/Azure-Sentinel/tree/master/DataConnectors#build-the-connector)